tsconfig.json

```
@@ -15,7 +15,7 @@
 		"allowJs": true
 	},
 	"include": [
-		"src/**/*"
+		"src/**/*",
```

This is invalid json
```suggestion
		"src/**/*"
```

---

Thanks!